enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/InteractiveQuestionProviderWithLocation.tsx

```diff
@@ -9,7 +9,9 @@ export const InteractiveQuestionProviderWithLocation = ({
   params,
   ...props
 }: InteractiveQuestionProviderWithLocationProps) => {
-  const cardId = Urls.extractEntityId(params.slug);
+  // If we cannot extract an entity ID from the slug, assume we are creating a new question.
+  const cardId = Urls.extractEntityId(params.slug) ?? "new";
```

Comment:
`Urls.extractEntityId` returns `number | undefined`, so we have to cast `undefined` to `new` as we no longer accepts undefined card ids.